src/components/validations/validationRules.ts

```diff
@@ -0,0 +1,25 @@
+import { baseRule } from '@/components/validations/baseRule'
+
+export const email: baseRule[] = [
+  {
+    required: true,
+    message: 'Email is required',
+    trigger: 'blur'
+  }
+]
+
+export const password: baseRule[] = [
+  {
+    required: true,
+    message: 'Password is required',
+    trigger: 'blur'
+  }
+]
+
+export const passwordConfirmation: baseRule[] = [
+  {
+    required: true,
+    message: 'Password confirmation is required',
+    trigger: 'blur'
+  }
+]
```

Comment:
Do you need to use Array<BaseRule> with single object inside, or can you just use for example
`email: baserule = { ... }`?